Apps/Database/src/Delegates/DBCommentDelegate.cs

```diff
@@ -59,6 +59,19 @@ public DBResult<IEnumerable<Comment>> GetList(string hdId, string parentEntryId)
             return result;
         }
 
+        /// <inheritdoc />
+        public DBResult<IEnumerable<Comment>> GetAll(string hdId)
```

Comment:
this are change that are already in the dev branch, we need to update the features/9722 branch from dev so it doesn't display these changes and we can check the actual change you did!

Reply:
yep, i did that, this is part of it